source/_integrations/webostv.markdown

```diff
@@ -34,6 +34,10 @@ To begin with enable *LG Connect Apps* feature in *Network* settings of the TV.
 
 Home Assistant is able to turn on an LG webOS Smart TV if you specify an action, provided by an {% term integration %} like [HDMI-CEC](/integrations/hdmi_cec/) or [WakeOnLan](/integrations/wake_on_lan/).
```

Comment:
```suggestion
If you want to use an automation to turn on an LG webOS Smart TV, install an {% term integration %} such as the [HDMI-CEC](/integrations/hdmi_cec/) or [WakeOnLan](/integrations/wake_on_lan/). They provide an action that can be used for that.
```

Reply:
Works for me. Although emphasizing and duplicating are not bad things per se when they help making things obvious.